libyul/backends/evm/EVMDialect.h

```diff
@@ -49,15 +50,17 @@ struct BuiltinFunctionForEVM: BuiltinFunction
  */
 struct EVMDialect: public Dialect
 {
-	EVMDialect(AsmFlavour _flavour, bool _objectAccess);
+	EVMDialect(AsmFlavour _flavour, bool _objectAccess, dev::solidity::EVMVersion _evmVersion);
```

Comment:
I was going to ask about a default value, but I guess it's better to enforce it.